src/main/java/com/pubnub/api/endpoints/objects_api/utils/Include.java

```diff
@@ -11,7 +11,7 @@
 
 public class Include implements ParameterEnricher {
     static final String INCLUDE_PARAM_NAME = "include";
-    static final String INCLUDE_CUSTOM_PARAM_VALUE = "custom";
+    static final String INCLUDE_CUSTOM_PARAM_VALUE = "custom,type,status";
```

Comment:
But why? Shouldn't this be handled by `ObjectApiEndpoint`?

Reply:
The thing is that method Include#enrichParameters is executed after ObjectApiEndpoint#createBaseParams. So even you set proper values in ObjectApiEndpoint#createBaseParams then they might be overritten in Include#enrichParameters.
I could revert the change you pointed out but then modification of  Include#enrichParameters is required from

from
    public Map<String, String> enrichParameters(Map<String, String> baseParams) {
        final Map<String, String> enrichedMap = new HashMap<>(baseParams);
        if (!inclusionFlags.isEmpty()) {
            enrichedMap.put(INCLUDE_PARAM_NAME, join(inclusionFlags));
        }
        return enrichedMap;
    }
to 
    public Map<String, String> enrichParameters(Map<String, String> baseParams) {
        final Map<String, String> enrichedMap = new HashMap<>(baseParams);
        if (!inclusionFlags.isEmpty()) {
            String currentIncludeValue = enrichedMap.get(INCLUDE_PARAM_NAME);
            inclusionFlags.add(currentIncludeValue);
            enrichedMap.put(INCLUDE_PARAM_NAME, join(inclusionFlags));
        }
        return enrichedMap;
    }

Do you see other options?

Reply:
Maybe remove change in `ObjectApiEndpoint` and that one here and put something like this:

```
static final Collection<String> DEFAULT_INCLUDE_FLAGS = Arrays.asList("type", "status");

private final List<String> inclusionFlags = new ArrayList<>(DEFAULT_INCLUDE_FLAGS);
```

That way you should have it regardless of custom being added or not. The question that I have for you though. Are you sure we should add always both `type` and `status`? I'm pretty sure it's a `status` only for memberships, but somehow I don't see it in Kotlin implementation so I don't know anymore

Reply:
As discussed on a cal I am working on this.